crates/rune-modules/src/http.rs

```diff
@@ -172,15 +1515,46 @@ impl Response {
     }
 
     /// Get the status code of the response.
-    #[rune::function]
+    #[rune::function(instance)]
     fn status(&self) -> StatusCode {
         let inner = self.response.status();
         StatusCode { inner }
     }
+
+    /// Get the version of the response.
+    #[rune::function(instance)]
+    fn version(&self) -> Version {
+        let inner = self.response.version();
+        Version { inner }
+    }
+
+    // TODO: response headers
+
+    /// Get the content-length of this response, if known.
+    ///
+    /// Reasons it may not be known:
+    ///
+    /// - The server didn't send a `content-length` header.
+    /// - The response is compressed and automatically decoded (thus changing
+    ///   the actual decoded length).
+    #[rune::function(instance)]
+    fn content_length(&self) -> Option<u64> {
+        self.response.content_length()
+    }
+
+    /* TODO: Make rune net module
```

Comment:
Please don't keep commented out code around, just remove it. If you want to keep the code around for the future, maintain it in a separate branch.